### ISSUE TITLE
bug: Phi-3 models failing to start with outdated versions

### ISSUE THREAD

**Issue by imtuyethan** [janhq/jan#3608]
## Goal

- Update `model.json` in Jan's original Model Folder structure to point to downloadable Phi3 model

## Out-of-scope

- We will handle proper Phi3.5 support in subsequent sprint #3461 

### Jan version

v0.5.3-623

### Describe the Bug

- Phi-3 models (Mini, Medium, 3B) might be outdated (from Microsoft, last update was 4 months ago.
- Some Phi-3 models fail to start, particularly Phi-3 Mini.
- There's an error loading model hyperparameters related to flash attention?

### Steps to Reproduce

1. Open Jan version 0.5.3-623
2. Attempt to use any Phi-3 model (Mini, Medium, or 3B)
3. Try to start a conversation with any of these models
4. Note that the model fails to start or respond weirdly

### Screenshots / Logs

https://github.com/user-attachments/assets/6cf7542b-63b8-45cf-b04b-5ba36a1cff2f

![Screenshot 2024-09-09 at 3 13 14 PM](https://github.com/user-attachments/assets/3f75c188-815e-4495-9b31-638ff6cbc8b0)

![scs](https://github.com/user-attachments/assets/90f410d3-90af-4d87-b993-1366616d2683)

### What is your OS?

- [X] MacOS
- [ ] Windows
- [ ] Linux

App logs: 
[app.log](https://github.com/user-attachments/files/16927695/app.log)

**Comment by imtuyethan** [comment]
Related issue was reported: https://discord.com/channels/1107178041848909847/1278704128905973873

**Comment by imtuyethan** [comment]
Proposed fix:
- Take these models out of Hub for now
- Add the latest versions

**Comment by 0xSage** [comment]
> Proposed fix:
> 
> * Take these models out of Hub for now.
> * Update model.json for the entire phi-3 group once the refactor is done.

1. Can we fix root issue rather than taking models out of hub? What's the root cause
2. model.json? Do you mean model.yaml?

**Comment by 0xSage** [comment]
Ok, it seems the issue is: 

1. We reverted back to the old method of listing models (by including the model.jsons)
2. Phi model form microsoft is incompatbile with llamacpp latest
3. We either: wait for cortex rebase or as a temp fix, fix the model URL.

I suggest: 
1. If next release doesn't update Hub to new cortexso hub, then let's just fix the model URL.
2. If next release does update the hub entirely, then this issue is already resolved

**Comment by dan-homebrew** [comment]
@louis-jan I am scoping this to just changing the Model URLs in `model.json` as a hotfix 
 
- We can handle proper Phi3.5 support later on with #3461 (with Built-in Models)

**Comment by louis-jan** [comment]
Copy that!

**Comment by imtuyethan** [comment]
Fix:
- Change URL for Phi-3 Mini
- Update Prompt Template for Phi-3 Medium 
- Phi-2 is old & deprecated, @imtuyethan should delete it
